app/routes/user.routes.js

```diff
@@ -18,7 +16,9 @@ const {
 } = require('../middlewares/authentication.middleware');
 
 // Post login request
-router.post('/login', checkLoginInput, session.login);
+router.post('/login', passport.authenticate('local'), (req, res, _next) =>
+  res.status(200).json(req.user.stripSensitive())
+);
```

Comment:
please do not put functions in route files, it goes against the entire architecture. relegate the `res.status(200).json(req.user.stripSensitive()` to a controller to be handled